internal/olm/client/client.go

```diff
@@ -113,17 +113,44 @@ func (c Client) DoCreate(ctx context.Context, objs ...client.Object) error {
 	for _, obj := range objs {
 		kind := obj.GetObjectKind().GroupVersionKind().Kind
 		log.Infof("  Creating %s %q", kind, getName(obj.GetNamespace(), obj.GetName()))
-		err := c.KubeClient.Create(ctx, obj)
-		if err != nil {
-			if !apierrors.IsAlreadyExists(err) {
+
+		for i := 0; i < 5; i++ { // try to create 5 times before giving up
```

Comment:
It looks like if we get the `NoMatchError` 5 times, we'll swallow the error and return nil. We probably would want to surface the error in this case instead of swallowing it.

Reply:
:+1: good point. But in that case, what do you say about making it a lop of ten, just for case?